setup.py

```diff
@@ -45,7 +45,9 @@ def get_version(rel_file='plugin.yaml'):
     install_requires=[
         "cloudify-common>=4.5.5",
         "cloudify-utilities-plugins-sdk==0.0.28",  # obfuscate_passwords
-        "pyyaml==3.12"
+        "pyyaml==3.12",
```

Comment:
Why should we use specifically 3.12?

Reply:
All our plugins are using this version. Its because old managers(packages issues)